### PR TITLE
B35 get/api/bookmark

### PR DESCRIPTION
**Pull request by garysum88**
In addition to necessary changes for GET Bookmark endpoints, I have also edited:

-model/location.go 
-routes/location.go

edited utils - location structs (add back the form instruction)

so that the GetLocation function can work again

### CODE REVIEW

File: routes/reviews.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package routes
 
 import (
 	"github.com/gofiber/fiber/v2"
+	"fmt"
```
Comment (jaykeHarrison) [inline routes/reviews.go:5]: imported and not used

File: routes/bookmark.go [hunk 2]
```diff
@@ -23,4 +25,31 @@ func PostBookmark (c *fiber.Ctx) error {
     
     return c.Status(200).JSON(responseBookmark)
 
+}
+
+
+func GetBookmarks(c *fiber.Ctx) error {
+
+
+
```
Comment (jaykeHarrison) [inline routes/bookmark.go:34]: while you're deleting fmt, delete some blank lines lol

**Review by jaykeHarrison — CHANGES_REQUESTED** [review]

File: main.go [hunk 3]
```diff
@@ -22,13 +22,15 @@ func setUpRoutes(app *fiber.App) {
...
+	app.Get("/api/bookmarks/:user_id", routes.GetBookmarks)
+  app.Get("/api/users/:user_id", routes.GetUserById)
+	app.Get("/api/locations/:location_id", routes.GetLocationById)
 	app.Post("/api/users", routes.PostUser)
 	app.Post("/api/reviews", routes.PostReview)
 	app.Post("/api/locations", routes.PostLocation)
 	app.Post("/api/bookmarks", routes.PostBookmark)
-	app.Get("/api/users/:user_id", routes.GetUserById)
-	app.Get("/api/locations/:location_id", routes.GetLocationById)
-}
```
Comment (jaykeHarrison) [inline main.go:31]: need to undelete this curly brace

**Review by jaykeHarrison — DISMISSED** [review]
awesome, everything is running now and getting the review :)

**Review by nmathew222 — DISMISSED** [review]

**Review by jaykeHarrison — DISMISSED** [review]

**Review by nmathew222 — APPROVED** [review]

**Pull request merged**